userdocs/userdocs.docc/Articles/Features/docc-live-preview.md

```diff
@@ -0,0 +1,11 @@
+# Documentation Live Preview
+
+Show a live preview of your Swift documentation while editing.
+
+> Note: This feature is only available when using a Swift toolchain 6.2 or higher running on macOS or Linux.
```

Comment:
```suggestion
```

Shift this down below, and use the availability marker for the Swift language version to indicate this is Swift 6.2+ only